kernel/Dockerfile.zfs

```diff
@@ -52,6 +52,9 @@ RUN ./autogen.sh && \
     make -j "$(getconf _NPROCESSORS_ONLN)" && \
     make install
 
+# Run depmod against the new module directory.
+RUN depmod -ae `ls /lib/modules`
```

Comment:
some shell script style issues:
1. can you use `$(...)` not backticks
2. can you use `find` not `ls`
Possibly you can just do `depmod -ae /lib/modules/*` though instead...

Reply:
depmod won't take a path, tried a few variants; thought `ls` was the most readable, happy to change.

```
Removing intermediate container 47e6e498c868
Step 20/25 : RUN depmod -ae /lib/modules/*
 ---> Running in 10e5ae3abf3a
depmod: can't change directory to 'lib/modules/4.9.41-moby': No such file or directory
The command '/bin/sh -c depmod -ae /lib/modules/*' returned a non-zero code: 1
```